build.gradle

```diff
@@ -133,19 +133,21 @@ subprojects {
     //        options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
     //    }
 
-    task javancss(dependsOn: 'classes') << {
-        ant {
-            logging.level = LogLevel.INFO
-            taskdef(name:'javancss', classname:'javancss.JavancssAntTask', classpath: configurations.javancss.asPath)
-            javancss(
-                srcdir:'src/main/java',
-                abortOnFail:'true',
-                generateReport:'true',
-                packageMetrics:'true',
-                classMetrics:'false',
-                functionMetrics:'false'
-                // outputFile: reportName,
-            )
+    task javancss(dependsOn: 'classes') {
+        doLast {
+            ant {
+                logging.level = LogLevel.INFO
+                taskdef(name: 'javancss', classname: 'javancss.JavancssAntTask', classpath: configurations.javancss.asPath)
+                javancss(
+                        srcdir: 'src/main/java',
+                        abortOnFail: 'true',
+                        generateReport: 'true',
+                        packageMetrics: 'true',
+                        classMetrics: 'false',
+                        functionMetrics: 'false'
+                        // outputFile: reportName,
```

Comment:
to remove ?

Reply:
Since i haven't commented it, i prefer to keep it like that